packages/blockly-field-date/package.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "@blockly/field-date",
+    "version": "3.20200304.0",
+    "description": "A Blockly date picker field that uses the Google Closure date picker.",
+    "scripts": {
```

Comment:
would be nice to also have a clean to clean up the built files.

Reply:
no need, built files are gitignored. The problem with doing something like ``rm -rf`` is its not cross platform. The alternative is to use an npm library (something like rimraf) and that just seems too much overhead to clear out a build directory.